### PR TITLE
Setup GitHub Pages for displaying oncall rotation

### PR DESCRIPTION
**Pull request by adrcunha**
Curious? See https://adrcunha.github.io/test-infra/

Part of #1099

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Current Oncall 
+
+<iframe src="https://storage.googleapis.com/knative-infra-oncall/oncall.html" width="100%" height="384"></iframe>
```
Comment (chaodaiG) [inline docs/README.md]: this html tag does seem to work in markdown file

Comment (adrcunha) [reply docs/README.md]: It does. Our linter doesn't like it:

```
docs/README.md:1: MD009 Trailing spaces
docs/README.md:3: MD033 Inline HTML

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
```

I'll disable the rule for our repo.

Comment (chaodaiG) [reply docs/README.md]: It still shows raw html in your fork, would it be different in knative?
https://github.com/adrcunha/test-infra/blob/onca/docs/README.md

Comment (adrcunha) [reply docs/README.md]: That's expected, as this markdown file is meant to be viewed through GitHub Pages: https://adrcunha.github.io/test-infra/

Comment (chaodaiG) [reply docs/README.md]: thanks, good to know

**Pull request merged**